Util.java

```diff
@@ -1,26 +1,36 @@
 package org.usfirst.frc4904.standard;
 
-
 import edu.wpi.first.hal.util.BoundaryException;
 
 /**
  * Common utilities
  */
 public class Util {
 	/**
-	 * A function for determining if floating point numbers are effectively zero.
-	 * Floating point arithmetic tends to introduce very small errors.
+	 * Returns true if {@code var} is less than {@code epsilon}. This is useful for
+	 * floating point numbers, which tend to introduce small errors when operated
+	 * on.
 	 *
-	 * @param var
-	 *        The floating point number you want to compare
-	 * @return
-	 * 		Whether or not it is within Double.MIN_VALUE of zero
+	 * @param value   The floating point number to be compared
+	 * @param epsilon The maximum magnitude of var such that it can be considered
+	 *                zero
+	 * @return Whether or not {@code var} is less than {@code epsilon}
 	 */
-	public static boolean isZero(double var, double epsilon) {
-		return Math.abs(var) < epsilon;
+	public static boolean isZero(double value, double epsilon) {
+		return Math.abs(value) < epsilon;
 	}
-	public static boolean isZero(double var) {
-		return isZero(var, Math.sqrt(Math.ulp(1.0)));
+
+	/**
+	 * Returns true if {@code var} is less than {@code epsilon}. This is useful for
+	 * floating point numbers, which tend to introduce small errors when operated
+	 * on.
+	 *
+	 * @param value The floating point number to be compared
+	 * @return Whether or not {@code var} is effectively zero, determined by the ULP
```

Comment:
Again, specifying the semantics of the heuristic used is not necessary because it shouldn't be relevant to the end user in this case.